### PR TITLE
add course link

### PR DESCRIPTION
**Pull request by casperdcl**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
...
 - DVC Live (DVC Metrics Logger) | [Python API](https://github.com/iterative/dvclive) | [docs](https://dvc.org/doc/dvclive)
 - TPI: Terraform Provider Iterative | [code](https://github.com/iterative/terraform-provider-iterative) | [website & docs](https://registry.terraform.io/providers/iterative/iterative/latest/docs) | [Python API](https://github.com/iterative/tpi)
-- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories)
+- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories) | [Free course & certification](http://learn.iterative.ai)
```
Comment (casperdcl) [inline profile/README.md:6]: > `Free course & certification`

should wording be updated?

/CC @iterative/cse @jendefig @flippedcoder

Comment (casperdcl) [inline profile/README.md]:
```suggestion
- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories) | [Free course & certification](https://learn.iterative.ai)
```

SSL

File: profile/README.md [hunk 2]
```diff
@@ -3,4 +3,4 @@
...
 - DVC Live (DVC Metrics Logger) | [Python API](https://github.com/iterative/dvclive) | [docs](https://dvc.org/doc/dvclive)
 - TPI: Terraform Provider Iterative | [code](https://github.com/iterative/terraform-provider-iterative) | [website & docs](https://registry.terraform.io/providers/iterative/iterative/latest/docs) | [Python API](https://github.com/iterative/tpi)
-- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories)
+- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/enterprise.svg"/> [awesome-iterative-projects](https://github.com/iterative/awesome-iterative-projects) | [example repos](https://github.com/iterative?q=topic%3Aexample&type=public&language=&sort=stargazers#org-repositories) | [Free course & certification](https://learn.iterative.ai)
```
Comment (casperdcl) [inline profile/README.md:6]: > `Free course & certification`

should wording be updated?

/CC @iterative/cse @jendefig @flippedcoder

Comment (alex000kim) [reply profile/README.md:6]: `course` -> `courses` OR `training`? 

So that we don't need to change it in the future.

Comment (jendefig) [reply profile/README.md:6]: It looks like we won't have a second course until end of Q3 at the earliest.  I think we can update when we do.  I feel like training implies more 1:1 interaction.

Comment (jendefig) [reply profile/README.md:6]: Also, thank you @casperdcl 🙏

**Pull request merged**